tests/rt_cascading.erl

```diff
@@ -1204,25 +1266,6 @@ timeout(MultiplyBy) ->
             N * MultiplyBy
     end.
 
-eunit(TestDef) ->
-    case eunit:test(TestDef, [verbose]) of
-        ok ->
-            pass;
-        error ->
-            exit(error),
-            fail
-    end.
-
-maybe_skip_teardown(TearDownFun) ->
-    fun(Arg) ->
-        case rt_config:config_or_os_env(skip_teardowns, undefined) of
-            undefined ->
-                TearDownFun(Arg);
-            _ ->
-                ok
-        end
-    end.
-
 wait_until_pending_count_zero(Nodes) ->
     WaitFun = fun() ->
         {Statuses, _} =  rpc:multicall(Nodes, riak_repl2_rtq, status, []),
```

Comment:
I guess I can only comment on the diff parts - in line 1273 below we could change this whole expression to be

```erlang
%% somewhere else...
all_true(true) -> true;
all_true(_) -> false.

%% then at line 1273
lists:all(fun all_true/1, Out)
``` 

instead of what's there.